dom0/sd-sys-vms.sls

```diff
@@ -107,14 +113,25 @@ sd-{{ sys_vm }}-fedora-version-halt-wait:
     - require:
       - cmd: dom0-install-fedora-template
 
+# Since sys-usb uses sd-fedora-dvm, we can only change the latter's template
+# when it's not running
+{% if sys_vm == 'sys-usb' %}
+  qvm.vm:
+    - name: sd-fedora-dvm
+    - prefs:
+      - template: {{ sd_supported_fedora_version }}
+    - require:
+      - cmd: sd-{{ sys_vm }}-fedora-version-halt-wait
+{% endif %}
+
 sd-{{ sys_vm }}-fedora-version-update:
   qvm.vm:
     - name: {{ sys_vm }}
     - prefs:
       - template: {{ sd_supported_fedora_template }}
     - require:
       - cmd: sd-{{ sys_vm }}-fedora-version-halt-wait
-{% if sd_supported_fedora_template.endswith("-dvm") %}
+{% if not dvm_needs_updating and sd_supported_fedora_template.endswith("-dvm") %}
```

Comment:
I'm confused by the `not dvm_needs_updating` addition here -- what's the purpose of this additional check?

That said, what _appears_ to be happening on my system is that this evaluates to true, but the `create-` block itself is never defined, because the VM is already present and the `{% if required_dispvm not in existing_vms %}` block evaluates to false.